src/NovelImageRect.cpp

```diff
@@ -10,9 +10,9 @@ namespace NovelRT {
 NovelImageRect::NovelImageRect(NovelLayeringService* layeringService,
                                const float& screenScale,
                                const GeoVector<float>& size,
-                               const std::string_view imageDir,
-                               const NovelCommonArgs& args,
-                               const GLuint programId,
+                               std::string_view imageDir,
+                               NovelCommonArgs& args,
```

Comment:
This is passed by reference. Please re-add the `const` qualifier here.

Reply:
oh, my bad. I'll re-add it.